parsl/monitoring/db_manager.py

```diff
@@ -236,6 +244,55 @@ class Block(Base):
             PrimaryKeyConstraint('run_id', 'block_id', 'executor_label', 'timestamp'),
         )
 
+    class Files(Base):
+        __tablename__ = FILES
+        file_name = Column('file_name', Text, index=True, nullable=False)
+        file_path = Column('file_path', Text, nullable=True)
+        file_id = Column('file_id', Text, index=True, nullable=False)
+        run_id = Column('run_id', Text, nullable=False)
+        task_id = Column('task_id', Integer, nullable=True)
+        try_id = Column('try_id', Integer, nullable=True)
+        timestamp = Column('timestamp', DateTime, nullable=True)
+        size = Column('size', BigInteger, nullable=True)
+        md5sum = Column('md5sum', Text, nullable=True)
+        sa.Index("files_task_run_id_idx", "task_run_id", "task_id", "try_id")
+        __table_args__ = (PrimaryKeyConstraint('file_id'),)
+
+    class Environment(Base):
```

Comment:
see https://github.com/Parsl/parsl/pull/3712/files for a note about why these particular fields?

Reply:
I added individual indexes to the columns, so no real need for a composite index, I don't see these databases getting into tens of millions of rows.